aip/classic/read_records.py

```diff
@@ -3,8 +3,8 @@
 import time
 import timeout_decorator
 
-from aip.libs import xmltodict
-from aip.libs import enforce_schema
+from aip.classic import xmltodict
```

Comment:
Since xmltodict 0.11.0 is now included in requirements, this should just be "import xmltodict", and the file aip/classic/xmltodict.py should be removed.